### PR TITLE
Feature checkout flow

### PR DESCRIPTION
**Pull request by vikrantrana1337**
Edit form handling and Checkout address

### CODE REVIEW

File: src/config/index.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ const config = {
     }
   };
   
-  let configs = config.production;
+  let configs = config.development;
```
Comment (GrgSaloni) [inline src/config/index.js:20]: revert

File: src/utils/index.js [hunk 2]
```diff
@@ -2,4 +2,26 @@ export { default as history } from "./history";
...
+  IFSC: '^[a-zA-Z]{4}0[0-9a-zA-Z]{6}',
+  ONLY_NUM: '^[0-9]*$',
+  ONLY_ALPHA: '^[A-Za-z]+$',
+  ALPHA_NUMERIC: '^[a-zA-Z0-9]',
+  ALPHA_SPACE: '^^[A-Za-z0-9 ]+$',
+  ADDRESS: '^[-a-zA-Z0-9, ]'
+};
+
+export const handleValidation = {
+  validate: function(validationType, value, minLength) {
```
Comment (GrgSaloni) [inline src/utils/index.js:22]: where are you using this func?

File: src/config/index.js (hunk 1 above)
Comment (vikrantrana1337) [reply src/config/index.js:20]: done

File: src/utils/index.js (hunk 2 above)
Comment (vikrantrana1337) [reply src/utils/index.js]: i use this code at the time of edit form on next screen  now i remove this code

**Pull request merged**